tests/js/spec/views/organizationIntegrations/integrationListDirectory.spec.jsx

```diff
@@ -0,0 +1,332 @@
+import React from 'react';
+
+import {Client} from 'app/api';
+import {mountWithTheme} from 'sentry-test/enzyme';
+import IntegrationListDirectory from 'app/views/organizationIntegrations/integrationListDirectory';
+
+const mockResponse = mocks => {
+  mocks.forEach(([url, body]) => {
+    return Client.addMockResponse({
+      url,
+      body,
+    });
+  });
+};
+
+describe('IntegrationListDirectory', function() {
+  beforeEach(function() {
+    Client.clearMockResponses();
+  });
+
+  const org = TestStubs.Organization();
+  const routerContext = TestStubs.routerContext();
+  let wrapper;
+
+  describe('Renders view', function() {
+    beforeEach(() => {
+      mockResponse([
+        [
+          `/organizations/${org.slug}/config/integrations/`,
+          {
+            providers: [
+              {
```

Comment:
@NisanthanNanthakumar can we move these to `tests/js/fixtures` - I think we have some there for github, it would make these tests a bit easier to read